cypress/e2e/adminCouponList.cy.ts

```
@@ -0,0 +1,214 @@
+        it('상태 현재 상태 클릭 시 요청 없음', () => {
+            cy.get('[data-cy="status-select"]').select('전체')
+            cy.get('[data-cy="status-select"]').should('have.value', 'ALL')
+        })
+
+        it('상태 다른 상태 클릭 시 리로드', () => {
+            cy.fixture('admin/coupon/couponList.json').then((couponList) => {
+                const filteredCouponListContent = couponList.content.filter((coupon: any) => coupon.status === 'IN_PROGRESS')
+                cy.intercept('GET', '/api/admin/couponByStatus?size=10&page=0&status=IN_PROGRESS', {
+                    statusCode: 200,
+                    body: {
+                        content: filteredCouponListContent,
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)